packages/@o3r/eslint-plugin/schematics/ng-update/v11/update-configs/update-configs.ts

```diff
@@ -0,0 +1,32 @@
+import type { Rule } from '@angular-devkit/schematics';
+import { basename } from 'node:path';
+
+/**
+ * Update Otter cms.json in an Angular Project
+ */
+export function updateEslintJsonFile(): Rule {
+  const configFilePattern = /^.?eslint(:?rc)?\..*/;
+  const toReplace = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention -- following eslint config name
+    'angular-template-recommended': /(['"](?:plugin:)?@o3r\/)template-recommended(['"])/g,
+    // eslint-disable-next-line @typescript-eslint/naming-convention -- following eslint config name
+    'monorepo-recommended': /(['"](?:plugin:)?@o3r\/)json-recommended(['"])/g
+  };
+
+  return (tree) => {
+    tree.visit((path) => {
```

Comment:
We may not use `tree.visit` because it visit all node_modules etc...
we have an helper in @o3r/schematics `findFilesInTree` that is not visiting these folders